bin/faucet/src/client.rs

```diff
@@ -316,6 +317,7 @@ fn build_transaction_arguments(
         &DISTRIBUTE_FUNGIBLE_ASSET_SCRIPT
             .replace("{recipient}", &recipient)
             .replace("{note_type}", &Felt::new(note_type as u64).to_string())
+            .replace("{aux}", &Felt::default().to_string())
```

Comment:
Might be best to do `note.metadata().aux().to_string()` here so that it comes from the created note, even if it's hardcoded there as well.

Reply:
Done!